src/GitVersionExe/Program.cs

```diff
@@ -108,6 +110,11 @@ static int VerifyArgumentsAndRun()
             return 0;
         }
 
+        private static void VerifyConfiguration(Arguments arguments, IFileSystem fileSystem)
+        {
+            var gitPreparer = new GitPreparer(arguments.TargetUrl, arguments.DynamicRepositoryLocation,arguments.Authentication,arguments.NoFetch,arguments.TargetPath);
```

Comment:
Nitpick, but can you please format this according to the ReSharper style settings? We at least want spaces after commas, but if the line is very long, it would be nice with line breaks after each argument and the arguments aligned.